modules/is_master/templates/build.sh.erb

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# Apply configurations
+export FACTER_profile=is_master
+puppet agent -vt
+
+# exit immediately if a command exits with a non-zero status
+set -e
+
+updated_templates=()
+
+copy_to_agent() {
+  cd <%= @distribution_path %>
+  rm <%= @product %>-<%= @product_version %>.zip
+  echo "Repackaging updated pack..."
+  zip -qr <%= @product %>-<%= @product_version %>.zip <%= @product %>-<%= @product_version %>/
+  echo "Copying updated pack to Agent files directory..."
+  cp <%= @product %>-<%= @product_version %>.zip <%= @puppet_env %>/modules/is/files/
+}
+
```

Comment:
An extra line here. Shall we remove that?

Reply:
Resolved https://github.com/wso2/puppet-is/pull/58/commits/676e2fd0997af89eac923f728deb820bc7fdd18e